website/client/src/components/memberDetailsNew.vue

```diff
@@ -0,0 +1,330 @@
+<template>
+  <div
+    class="member-details-new"
+    :class="{ condensed, expanded }"
+    @click="showMemberModal(member)"
+  >
+    <div>
+      <avatar
+        :member="member"
+        :hide-class-badge="true"
+        @click.native="$emit('click')"
+        @mouseover.native="$emit('onHover')"
+        @mouseout.native="$emit('onHover')"
+      />
+    </div>
+    <div
+      class="member-stats"
+    >
+      <div class="d-flex align-items-center profile-first-row">
+        <class-badge
+          v-if="classBadgePosition === 'next-to-name'"
+          :member-class="member.stats.class"
+        />
+        <div class="d-flex flex-column profile-name-character">
+          <h3 class="character-name mt-75">
+            <user-link
+              :user-id="member._id"
+              :name="member.profile.name"
+              :backer="member.backer"
+              :contributor="member.contributor"
+              :smallerStyle="true"
+            />
+            <inline-class-badge
+              class="inline-class-badge"
+              v-if="member.stats"
+              :member-class="member.stats.class"
+            />
+          </h3>
+          <div class="small-text character-level">
+            <span
+              v-if="member.auth && member.auth.local && member.auth.local.username"
+              class="mr-1"
+            >@{{ member.auth.local.username }}</span>
+            <span
+              v-if="member.auth && member.auth.local && member.auth.local.username"
+              class="mr-1"
+            >•</span>
+            <span>{{ characterLevel }}</span>
+          </div>
+        </div>
+      </div>
+      <stats-bar
+        class="mt-3 stats-bar"
+        :icon="icons.health"
+        :value="member.stats.hp"
+        :max-value="MAX_HEALTH"
+        :tooltip="$t('health')"
+        progress-class="bg-health-new"
+        :condensed="condensed"
+        :show-icon="false"
+        :show-numbers="false"
+      />
+      <stats-bar
+        class="mt-75 stats-bar"
+        :icon="icons.experience"
+        :value="member.stats.exp"
+        :max-value="toNextLevel"
+        :tooltip="$t('experience')"
+        progress-class="bg-experience-new"
+        :condensed="condensed"
+        :show-icon="false"
+        :show-numbers="false"
+      />
+      <stats-bar
+        class="mt-75 stats-bar"
+        v-if="hasClass"
+        :icon="icons.mana"
+        :value="member.stats.mp"
+        :max-value="maxMP"
+        :tooltip="$t('mana')"
+        progress-class="bg-mana-new"
+        :condensed="condensed"
+        :show-icon="false"
+        :show-numbers="false"
+      />
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+  @import '~@/assets/scss/colors.scss';
+
+  .stats-bar {
+    margin-left: 0 !important;
```

Comment:
What's overriding this margin such that we need an `!important`?

Reply:
I wonder that myself now :D - maybe the copied version of the memberDetail had it ?  yeah can be removed